### PR TITLE
Made header show logged in username, put logout button in a dropdown

### PR DESCRIPTION
**Pull request by WillyST98**

### CODE REVIEW

File: Online-Messaging-App/src/app/home/header/header.component.html [hunk 1]
```diff
@@ -1,8 +1,16 @@
 <header>
     <mat-toolbar class="navbar">
         <div>Streamline Athletes</div>
+
         <div *ngIf="userLoggedIn">
+            <div class="headInfo">Welcome {{user[0].username}}!</div>
```
Comment (dBaiko) [inline Online-Messaging-App/src/app/home/header/header.component.html]: Remove the "Welcome". Also, if you include `private auth: AuthenticationService` in the constructor, you can simply call `this.auth.getAuthenticatedUser().getUsername()` here, no need to pull in via an http request.

File: Online-Messaging-App/src/app/home/header/header.component.ts [hunk 2]
```diff
@@ -8,11 +10,26 @@ import { AuthenticationService } from "../../shared/authentication.service";
...
+    constructor(private auth: AuthenticationService, private http: HttpClient) {
         this.userLoggedIn = auth.isLoggedIn();
     }
 
     ngOnInit(): void {
+        if (this.userLoggedIn==true)
+        this.getUserInfo();
+    }
+
+    getUserInfo(): void {
```
Comment (dBaiko) [inline Online-Messaging-App/src/app/home/header/header.component.ts]: Ya you really don't need this. The current username is always available in memory, just need to get it from the authentication service

File: Online-Messaging-App/src/app/home/logout/logout-form.component.scss [hunk 3]
```diff
@@ -0,0 +1,2 @@
+a {
```
Comment (dBaiko) [inline Online-Messaging-App/src/app/home/logout/logout-form.component.scss]: Does this serve a purpose

**Review by dBaiko — CHANGES_REQUESTED** [review]

File: Online-Messaging-App/src/app/home/header/header.component.html (hunk 1 above)
Comment (WillyST98) [reply Online-Messaging-App/src/app/home/header/header.component.html]: done

File: Online-Messaging-App/src/app/home/header/header.component.ts (hunk 2 above)
Comment (WillyST98) [reply Online-Messaging-App/src/app/home/header/header.component.ts]: done

File: Online-Messaging-App/src/app/home/logout/logout-form.component.scss (hunk 3 above)
Comment (WillyST98) [reply Online-Messaging-App/src/app/home/logout/logout-form.component.scss]: no already removed it on my end. I don't know why its showing up in the push. I double-checked that its gone now but if its still there (for some goddamned reason) just delete it and it should be fine

**Review by dBaiko — APPROVED** [review]

**Pull request merged**